wpeverest/user-registration
# Enhance - Select all option in multiple choice field

### All Submissions:






### Changes proposed in this Pull Request:
Previously, There was no select all option in the multiple-choice field. This PR provides the select all option in multiple choice field.



### How to test the changes in this Pull Request:

1. Navigate to the form builder and drag and drop the checkbox field.
2. In the Advance setting of field options of checkbox field there will be an option labelled a select all, enable the option.
3. Navigate to the registration form and verify whether the select all option is available or not. 

### Dependent PR:
https://github.com/wpeverest/user-registration-advanced-fields/pull/25

### Types of changes:




### Other information:




### Changelog entry

>Enhance - Select all option in multiple choice field
